gender_novels/novel.py

```diff
@@ -142,6 +143,33 @@ def find_quoted_text(self):
 
         return quotes
 
+    def get_count(self, word):
```

Comment:
Can you give the function a slightly more descriptive name, e.g. get_count_of_word(). That way we'll know what we're getting a count of.